docs/data/migration/migration-data-grid-v6/migration-data-grid-v6.md

```diff
@@ -14,15 +14,15 @@ To get started, check out [the blog post about the release of MUI X v6](https:/
 
 ## Start using the new release
 
-In `package.json`, change the version of the data grid package to `next`.
+In `package.json`, change the version of the data grid package to `7.x.x`.
```

Comment:
This will conflict with #12498. 🤔 
Consider aligning.
Given the previous migration file, it looks like going with `7.0.0` (like in the linked PR) is more appropriate. 🤔 🤷

Reply:
Thank you for pointing out, indeed the mentioned PR has a better version.
Reverted!